charts/nd-common/templates/_service.tpl

```diff
@@ -18,6 +18,8 @@ metadata:
   name: {{ default (include "nd-common.fullname" $) .Values.service.name }}
   labels:
     {{- include "nd-common.labels" $ | nindent 4 }}
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags: kubernetes_namespace={{ .Releases.Namespace }}
```

Comment:
I'm concerned that this implies that our `Service` objects are used to create load balancers in AWS .. but we don't do that anywhere. What problem are we trying to solve here?

Reply:
So this helper allows any type of service to be constructed. So we should add this annotation just in case someone does use Service.type=LB.